src/devices/localization2DServer/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+yarp_prepare_plugin(localization2DServer
+                    CATEGORY device
+                    TYPE localization2DServer
+                    INCLUDE localization2DServer.h
+                    DEFAULT ON)
+
+if(NOT SKIP_localization2DServer)
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+yarp_add_plugin(yarp_localization2DServer localization2DServer.h
+                                   localization2DServer.cpp
+                                  ${ROS_MSG})
```

Comment:
Is `${ROS_MSG}` actually used or it is a leftover?
In the case it used it should be specified among the dependencies

Reply:
used